udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -103,8 +103,9 @@ <h1 class="fr-mb-5v fr-h3">
             </h1>
             <h2 id="description" class="subtitle fr-mb-1w">{{ _('Description') }}</h2>
             <read-more max-height="description" class="fr-mb-n1w">
-                {{ dataset.description|markdown }}
+                <markdown-viewer data-content="{{dataset.description|to_json}}"></markdown-viewer>
```

Comment:
If we keep this idea, we should apply the same logic for other markdown contents? In particular, I'm thinking of *pages*.

Reply:
This is indeed the idea to have a unified viewer for all contents. This could allow us to use DSFR tables everywhere for example instead of our custom ones.